backend/src/main/java/de/neuefische/readandmeet/backend/controller/MeetingController.java

```diff
@@ -36,4 +36,10 @@ public Meeting getMeetingById(@PathVariable String id) {
     public void deleteById(@PathVariable String id) {
         this.meetingService.delete(id);
     }
+
+    @PutMapping("/{id}")
+    public Meeting updateMeetingById(@PathVariable String id, @RequestBody MeetingWithoutId meetingWithoutId) {
+        this.meetingService.editMeetingById(meetingWithoutId, id);
+        return meetingService.getDetails(id);
```

Comment:
```suggestion
    public Meeting updateMeetingById(@PathVariable String id, @RequestBody MeetingWithoutId meetingWithoutId) {
    try {
        this.meetingService.editMeetingById(meetingWithoutId, id);
        return ResponseEntity.ok(meetingService.getDetails(id))
        } catch (NoSuchElementException e) {
        return ResponseEntity.notFound().build();
```
If you use exceptions in your service, you can consider catching the Exceptions in your controller. 
It's only a small suggestion :)